shared-helpers/src/views/sign-in/FormSignIn.tsx

```diff
@@ -34,66 +35,136 @@ const FormSignIn = ({
     window.scrollTo(0, 0)
   }
   const { LinkComponent } = useContext(NavigationContext)
-
   return (
-    <FormCard>
-      <div className="form-card__lead text-center">
-        <Icon size="2xl" symbol="profile" />
-        <h1 className="form-card__title">{t(`nav.signIn`)}</h1>
-      </div>
-      <FormSignInErrorBox
-        errors={errors}
-        networkStatus={networkStatus}
-        errorMessageId={"main-sign-in"}
-      />
-      <div className="form-card__group pt-0">
-        <Form id="sign-in" className="mt-10" onSubmit={handleSubmit(onSubmit, onError)}>
-          <Field
-            caps={true}
-            name="email"
-            label={t("t.email")}
-            validation={{ required: true }}
-            error={errors.email}
-            errorMessage={t("authentication.signIn.enterLoginEmail")}
-            register={register}
-            dataTestId="sign-in-email-field"
+    <Card>
+      {process.env.showMandatedAccounts ? (
+        <>
+          <div className="form-card__lead mx-12 space-y-3 pl-0 pt-8">
+            <Icon size="2xl" symbol="profile" />
+            <Heading size="2xl" className="font-semibold">
+              {t("nav.signInLowercase")}
+            </Heading>
+          </div>
+          <FormSignInErrorBox
+            errors={errors}
+            networkStatus={networkStatus}
+            errorMessageId={"main-sign-in"}
+            className="p-0 mx-12"
           />
+          <div className="form-card__group mx-4">
```

Comment:
Same comment from the other PR that uptaking the seeds Card will require removing these and using sections, recommend checking out some of the examples :)